ci: use new Sprocket doc (#232)
stjudecloud/workflows: assets/DOCS.md

<p align="center">
  <a href="https://github.com/stjudecloud/workflows"><img src="./assets/workflows-banner-flowchart.jpg" width="800" title="St. Jude Cloud Workflows"></a>
  <br />
  <a href="https://github.com/stjudecloud/workflows/blob/master/LICENSE.md" target="_blank">
    <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-yellow.svg" />
  </a>
</p>

> This repository contains all bioinformatics workflows used on the St. Jude Cloud project. Officially, the repository is in beta — the project is adding workflows as they are developed and put into production.

> Resources requirements have been optimized to minimize failures in our computing environment, but they may not reflect the best settings for your use case. Please ensure that you tailor these parameters to fit your needs.

Reviewer: ```suggestion
Resources requirements and task parallelization have been optimized to minimize failures, but they may not reflect the best settings for your use case. Please ensure that you tailor these parameters to fit your needs.
```
Maybe? In particular thinking of the `after` clauses added in #235
